packages/firestore/CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Unreleased
 - [changed] Removed eval()-based fallback for JSON parsing, allowing SDK to
   be used in environments that prohibit eval().
+- [feature] Added a garbage collection process to on-disk persistence that
+  removes older documents. This is enabled by default, and the SDK will attempt
```

Comment:
"by default" => "automatically if persistence is enabled" (since persistence is off by default on web)?